src/main/java/org/truffleruby/parser/parser/RubyParser.y

```diff
@@ -2752,6 +2757,8 @@ assoc           : arg_value tASSOC arg_value {
                 | tDSTAR arg_value {
                     $$ = support.createKeyValue(null, $2);
                 }
+                | tDSTAR {
+                }
```

Comment:
What's this one for?

Reply:
I'm sorry. I thought I had pushed over that. MRI has a [`tDSTAR` rule](https://github.com/ruby/ruby/blob/31a5586d1e007d04cfa10548d9bfb42b2787a7a0/parse.y#L5726-L5736) that I was copying over before realizing that it's only on _master_ an not needed for the `Hash` punning feature.

Reply:
I've amended my commit and pushed over it.